### DISCUSSION TITLE
Downloading chia blockchain too slow

### DISCUSSION THREAD

**Discussion by vladimir-velinov** [Chia-Network/chia-blockchain#4894] [Support]
Hi,

I have setup chia on RaspberryPI 4, and it takes more than 3 days to sync the complete blockchain and it's still not complete. This is really frustrating. When I list the peer connections there are a lot of them, like 20+, but that doesn't seem to help the syncing process.
Can I download the blockchain as zip file and sync only the latest diff? This is available for some other blockchains like Verge(XVG).

Any advice is appreciated.
cheers,
 Vladimir

**Comment by SoraMakes** [comment]
stop the chia nodes and copy
    ~/.chia/mainnet/db/blockchain_v1_mainnet.sqlite
    ~/.chia/mainnet/wallet/db/blockchain_wallet_v1_mainnet_*.sqlite

See: https://github.com/Chia-Network/chia-blockchain/discussions/3185

**Reply by FSys77** [reply]
I usually do this.

Recently I encountered a problem when I copied the blockchain sqlite from my Raspberry to my Windows machine. My new Windows Chia installation was not able to work this this sqlite file.

Question: Are the sqlite files in Windows different from those on Linux?

**Comment by vladimir-velinov** [comment]
Thanks. This works in case I have previous copy, which I don't. The sd card got corrupt and I had to start all over. My question was in direction if chia would offer daily updated zip of the blockchain so that someone could download and then just sync the diff. Do you consider this in future? Because of the exponential growth currently I could assume that some nodes are becoming bottlenecks. Not sure if p2p connections are the problem for the fast blockchain or something else?

**Comment by vladimir-velinov** [comment]
So do you plan to have daily archive with the blockchain? This will reduce the traffic for newly joining nodes.

**Reply by itsChris** [reply]
https://chiazero.com/pages/chia-network-snapshot

not sure if it's a good idea to use that snapshot .. but it's worth a try.

**Comment by j4r3kb** [comment]
https://www.reddit.com/r/chia/comments/nkl5az/chia_blockchain_snapshot_bootstrap/
